src/vcpkg/base/system.process.cpp

```diff
@@ -24,7 +25,11 @@ extern char** environ;
 #endif
 
 #if defined(_WIN32)
+#include <Psapi.h>
+// #include <winternl.h>
+#include <TlHelp32.h>
 #pragma comment(lib, "Advapi32")
+// #pragma comment(lib, "Ntdll")
```

Comment:
We should avoid checking in commented out code. Assuming you want to keep the toolhelp scheme rather than going back to OpenProcess:

```suggestion
#include <TlHelp32.h>
#pragma comment(lib, "Advapi32")
```